packages/plugin-ext/src/main/node/paths/plugin-paths-service.ts

```diff
@@ -113,7 +114,7 @@ export class PluginPathsServiceImpl implements PluginPathsService {
         return path.join(
             homeDir,
             ...(isWindows ? this.windowsDataFolders : ['']),
```

Comment:
btw why do we need it on windows? Did it come from VS Code somehow?

Reply:
That has been done because creating `.theia` in user home on Windows is not that good idea. So, we used dedicated folders for this.

Reply:
Could you elaborate why Roaming is better? Should we do it generically for Windows fro all user data? What VS Code does?

I’ve noticed that at time when it was introduced the plugin system was not working well in windows, wonder whether it always tested or copied form somewhere.

Reply:
I am not the author of that code, but I do not think that is was copied.

VSCode has its data into `C:\Users\<username>\AppData\Roaming\Code` and also it creates `.vscode` in the user home (seems unpacked extensions are there). So uses both.